docs/contract/timeseries.md

```diff
@@ -6,6 +6,7 @@
 - The Time field(s):
   - Should have no null values
   - Field name is for display purposes only, there should be no labels
+  - Per each frame, any additional time fields after the first time field are treated as remainder data
```

Comment:
```suggestion
  - Must be the first field in the frame (frame.Fields[0]).  Any additional time fields are treated as remainder data
```

maybe 🤷

Reply:
I don't think it should need to be the first field, are you aware of anything that requires that currently?

Reply:
the timeseries panel moves the first time field to the first field.  

I can't think of any reason the time field should *not* be the first field in a timeseries.  The cases I have seen where it is not first, deciding if the dataset is timeseries or not is really a judgement call (SQL queries that happen to have a time value)

Reply:
Going to keep it as is for now. This behavior has been my underlying assumption. I don't think it matters that much as long as there are rules and don't want to audit what is out there to see if this would add more breaking changes.